docs/alerts/understanding-alerts/what-is-a-netdata-alert.md

```diff
@@ -0,0 +1,99 @@
+- Visible in the local Agent dashboard
+- Queryable via Agent APIs (`/api/v1/alarms`, `/api/v1/alarm_log`)
+- Sent to Netdata Cloud and displayed in the Events Tab
+- Used to trigger notifications (email, Slack, PagerDuty, etc.)
+
+## Alert Statuses
+
+Every alert has one of six possible statuses at any given time:
+
+| Status | Meaning |
+|--------|---------|
+| **UNINITIALIZED** | The alert has just been created but has not yet collected enough data to evaluate |
+| **UNDEFINED** | The alert could not be evaluated (missing data, division by zero, NaN/Inf result) |
+| **CLEAR** | The alert can be evaluated and neither warning nor critical thresholds are triggered; the metric is in a normal, healthy state |
+| **WARNING** | The warning expression evaluated to true; something may require attention |
+| **CRITICAL** | The critical expression evaluated to true; if both warning and critical conditions are met, CRITICAL takes precedence |
+| **REMOVED** | The alert has been deleted (configuration reload, child node disconnected, or agent shutdown) |
+
+### Status Lifecycle
+
+Alerts typically flow through this lifecycle:
+
+```
+UNINITIALIZED → UNDEFINED / CLEAR → WARNING → CRITICAL
+```
+
+- New alerts start in `UNINITIALIZED` until enough data is collected
+- Once data is available, they move to `CLEAR` (healthy) or `UNDEFINED` (cannot evaluate)
+- From `CLEAR`, they can transition to `WARNING` or `CRITICAL` when conditions are met
+- They return to `CLEAR` when conditions are no longer met
+- `REMOVED` appears when an alert is deleted during configuration reload or agent shutdown
+
+## How Alerts Relate to Charts and Contexts
+
+Netdata alerts inspect **metrics from charts**. Each chart represents a specific metric or set of related dimensions (for example, CPU usage, disk I/O, network traffic).
+
+There are two ways alerts can be attached to charts:
+
+### 1. Chart-Specific Alerts (Alarms)
+- An **alarm** is attached to a **single instance** of a chart
+- Example: "Alert if `eth0` network interface bandwidth > 90%"
+- Use when you need a rule for one specific chart
+
+### 2. Context-Based Alerts (Templates)
+- A **template** applies the same alert logic to **all charts that match a context**
+- Example: "Alert if *any* network interface bandwidth > 90%"
+- Use when you want the same rule to monitor all instances of a type (all disks, all network interfaces, all containers, etc.)
+
+**The difference:**
+- **Alarm** = one rule → one chart
+- **Template** = one rule → all matching charts
+
+Both types inspect metric values from charts in the same way; the difference is in **scope** (specific vs. all matching instances).
+
+:::note
+
+Detailed syntax for defining alarms vs templates is covered in **1.2 Alert Types: `alarm` vs `template`** and **Chapter 3: Alert Configuration Syntax**.
+
+:::
+
+## Where Alert Evaluation Happens
+
+As established in the chapter introduction:
+
+- Alerts are **evaluated locally** on each Netdata Agent or Parent node
```

Comment:
Let's just focus on Agent. The term "Parent node" is confusing, because "node" is the logical construct of a machine for which the metrics are being observed, whereas an Agent is a running piece of software. An Agent is the thing doing the evaluating on behalf of one or more nodes. Every Agent has a node that represents "itself".